### ISSUE TITLE
Configurable Save Locations

### ISSUE THREAD

**Issue by Baa14453** [Douile/image-saver#2]
Not sure if you're interested but it would be really useful if I could add one-click save locations to the context menu.

Like I could add a list of folders and each one would add a context menu entry that would save the image directly into the configured folder.

**Comment by Douile** [comment]
I don't think it is possible to change the download directory with the current webExtension API. https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/downloads/download
